package.json

```diff
@@ -88,6 +88,7 @@
     "mime": "^3.0.0",
     "ncp": "^2.0.0",
     "node-stream-zip": "^1.15.0",
+    "printj": "^1.3.1",
```

Comment:
I'm not comfortable bringing it as a dependency, we should either perform a clean room implementation or pull it as a third party. a new implementation might be better.